nixpkgs.nix

```diff
@@ -1,6 +1,6 @@
 let
   nixpkgs = builtins.fetchTarball {
-    url = "https://github.com/NixOS/nixpkgs/archive/db858b4d3032aec35be7e98a65eb9b91b63671ef.tar.gz";
-    sha256 = "0gqcbf5nyqff1a4ps6szcrv59ay97fr26jdwrs7qp8fijzcpdnkh";
+    url = "https://github.com/NixOS/nixpkgs/archive/19.09.tar.gz";
```

Comment:
Should we pin this to a particular commit, from 19.09 or latest master, rather than latest of 19.09? Because the 19.09 branch gets updated frequently and the sha will change when it happens.

Reply:
This is a GitHub [release](https://github.com/NixOS/nixpkgs/releases), not a branch. My understanding is that it can only change if somebody actually deletes and then recreates the release.

This said, I'm not sure what is the best practice here. Maybe a commit hash + a comment that it corresponds to the 19.09 release?

Reply:
I'd rather not use latest master because then all projects would end up on different nixpkgs versions needlessly.

Reply:
People usually use [niv](https://github.com/nmattia/niv) to manage versions is my understanding. I personally feel it's a bit too much for our use here.
And I realize now that you are right, that it's a release and should not change. So it should be fine. Also yes, 19.09 seems better rather than master as all projects will be on the same versions. :+1: